tests/tools_tests/BUILD

```diff
@@ -16,4 +16,8 @@ sh_test(
         "@bazel_tools//tools/bash/runfiles",
         "@cgrindel_bazel_starlib//shlib/lib:assertions",
     ],
+    target_compatible_with = select({
```

Comment:
The `shasum` command is not available on Windows. I think just running this test on Linux/MacOS is enough.